languages/swift/exercises/concept/closures/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+In this exercise you will be using closures with a higher-order function to generate sequences of numbers from a seed number.
```

Comment:
There might be a little _too_ much content in these instructions. Is there anything in particular in tasks 5 and 6 that the student has not had to deal with in the previous 4 tasks? If not, I would suggest removing tasks 5 and 6.